templates/limit/templates.yaml

```diff
@@ -0,0 +1,147 @@
+dataset: limit
+templates:
+  0898caf1-f3e4-493f-a838-84a678176c14: !Template
+    id: 0898caf1-f3e4-493f-a838-84a678176c14
+    jinja: '{{sentence}}
+
+
+      What is the last moving entity mentioned in the sentence if any?
+
+      |||
+
+      {% if (motion_entities  | length) > 0 %}
+
+      {{ (motion_entities | sort(attribute="start_index") | last)["entity"] }}
+
+      {% endif %}'
+    name: last_moving_entity
+    reference: ''
+    task_template: false
+  3b88c578-db77-4fd0-ad50-c78a39197ce5: !Template
+    id: 3b88c578-db77-4fd0-ad50-c78a39197ce5
+    jinja: '{{sentence}}
+
+
+      Is there any motion entities in the sentence?
+
+
+      |||
+
+
+      {{motion}}'
+    name: any_entity
+    reference: Asking if there is any entity in motion in the text
+    task_template: false
+  3f1689a9-b255-4d8d-b780-062ca2f83596: !Template
+    id: 3f1689a9-b255-4d8d-b780-062ca2f83596
+    jinja: "{{sentence}}\n\nWhat are the moving entities in the previous sentence?\
+      \ Return {{\"'No entity'\"}} if you can't find any. \n\n|||\n{% if (motion_entities\
+      \  | length) == 0 %}\n{{ \"No entity\" }}\n{% else %}\n{{motion_entities | map(attribute=\"\
+      entity\") | join(\", \")}}\n{% endif %}"
+    name: find_entities_question
+    reference: ''
+    task_template: true
+  74c9962e-3ec2-4f06-ace4-fcac6f506076: !Template
+    id: 74c9962e-3ec2-4f06-ace4-fcac6f506076
+    jinja: 'Extract: {{sentence}}
+
+
+      Is there more than one mention of a moving entity in the extract?
+
+
+      |||
+
+      {% if (motion_entities  | length) > 1 %}
+
+      {{ "Yes" }}
+
+      {% else %}
+
+      {{ "No" }}
+
+      {% endif %}'
+    name: more_than_one
+    reference: ''
+    task_template: false
+  766ab346-6fa6-4496-915f-65e7b06ab8ac: !Template
+    id: 766ab346-6fa6-4496-915f-65e7b06ab8ac
+    jinja: '{{sentence}}
+
+
+      How many moving entities are mentioned in the sentence?
+
+
+      |||
+
+
+      {{motion_entities | length}}'
+    name: count_entities
+    reference: ''
+    task_template: false
+  957deab1-7570-4cbf-a31f-55bfad5212a7: !Template
+    id: 957deab1-7570-4cbf-a31f-55bfad5212a7
+    jinja: "Name the motion entities in the following sentence. Respond {{\"'No entity'\"\
+      }} if you can't find any. \n\n{{sentence}}\n\n|||\n\n{% if (motion_entities\
+      \ | length) == 0 %}\n{{\"No entity\"}}\n{% else %}\n{{motion_entities | map(attribute=\"\
+      entity\") | join(\", \")}}\n{% endif %}"
+    name: find_entities_affirm
+    reference: ''
+    task_template: true
+  af2203ba-d176-4981-82bd-088ef0c39214: !Template
+    id: af2203ba-d176-4981-82bd-088ef0c39214
+    jinja: '{{sentence}}
+
+
+      Name the first motion entity mentioned in the sentence if any
+
+
+      |||
+
+
+      {% if (motion_entities  | length) > 0 %}
+
+      {{ (motion_entities | sort(attribute="start_index") | first)["entity"] }}
+
+      {% endif %}'
```

Comment:
add else condition to say "no entities in motion" if none

Reply:
Done